### DISCUSSION TITLE
How to disable export wizard from popping up if you use @export preprocessing command

### DISCUSSION THREAD

**Discussion by danielmumo** [dbeaver/dbeaver#20406] [Q&A]
Hello, I regularly run and export db data using a few repeated queries. I have managed to declare the export parameters using the @export command but each time I run the query, the export wizard pops up prefilled with all the options I just set and I have to click next next next proceed for the process to begin. This is super annoying and I would like to disable it. If you have any ideas on how to disable it please share.

My export parameters are:

`@export {"type": "csv", "producer": {"openNewConnections": "true", "queryRowCount": "true"}, "consumer": {"outputFolder": "C:/Users/dnk/Documents/My_Data_Dumps", "outputFilePattern": "${scriptFilename}_${timestamp}", "outputTimestampPattern": "dd-MM-yyyy"}, "processor": {}}`

**Comment by E1izabeth** [comment]
It sounds like a good improvement. Could you create a feature request?

Also you could try [DBeaver Tasks](https://dbeaver.com/docs/wiki/Task-Management/).
